src/standalone/Snow/Snow.jl

```diff
@@ -136,8 +136,8 @@ function ZenithAngleAlbedoModel(
     α_0::FT,
     Δα::FT,
     k::FT;
-    β = FT(0),
-    x0 = FT(0.2),
+    β = FT(0.7875), # Calibrated value
```

Comment:
I think we can change the constructor to supply defaults for all parameters now, and then use this in the long run.